s3/pkg/datastore/ceph/ceph.go

```diff
@@ -310,6 +310,10 @@ func (ad *CephAdapter) ListParts(ctx context.Context, multipartUpload *pb.ListPa
 	return nil, ErrNotImplemented
 }
 
+func (ad *CephAdapter) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3) error {
+	return nil
```

Comment:
@vineela1999 nil means success, right ?
do you want to distinguish between nil(successful check) and not-implemented instead ?

Reply:
yes changed